hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaRolloutManagement.java

```diff
@@ -1139,12 +1138,10 @@ private void assertTargetsPerRolloutGroupQuota(final long requested) {
      *
      * @param target
      *            The target
-     * @param requested
-     *            number of actions to check
      */
-    private void assertActionsPerTargetQuota(final Target target, final int requested) {
+    private void assertActionsPerTargetQuota(final Target target) {
```

Comment:
I would rename the method, so that is clear we are checking only one additional Action quota